README.md

```diff
@@ -4,9 +4,25 @@
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 ## Usage
-```
-npm install @smg-automotive/example
-```
+
+1. Install the package
+   ```sh
+   npm install @smg-automotive/components
+   ```
+2. run the setup script
+   ```sh
+   npx run components setup --path=<path to you public dir>
```

Comment:
Don't we need to use `@smg-automotive/components` when running the package command with npx?

Reply:
it depends. If we have it installed and listed in dependencies (which the setup assumes), then no.
If you would want to install the package then you'd need run it with the package:
```
npx -p @smg-automotive/components -c components setup
```